backend/.env.example

```diff
@@ -0,0 +1,5 @@
+# Env variables
+NODE_URL=#
+# If network == `mainnet` it will use StarknetChainId.SN_MAIN
+# otherwise it will use by default StarknetChainId.SN_SEPOLIA
+NETWORK=#
```

Comment:
For the example, I would put in a public NODE_URL such as https://free-rpc.nethermind.io/sepolia-juno

Reply:
1. I tested it with mainnet for `all events` - ok 
2. I tested it with sepolia for `specific address` - ok